### PR TITLE
Fix hero background overlay

### PR DESCRIPTION
**Pull request by Jonnie-Dev**
Before
image

After
image

### CODE REVIEW

**Comment by saenuruki** [comment]
@Jonnie-Dev Thanks!! Could you add the image of before and after?

**Comment by Jonnie-Dev** [comment]
Can you help with the before image, messed up my branches here

**Review by saenuruki — APPROVED** [review]
Seems good! Thanks!!

File: components/Header/Header.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Header = () => {
 	return (
 		<>
 			<div className="header-placeholder"></div>
-			<div className="header with-placeholder">
+			<div className="header with-placeholder bg-dark-06">
```
Comment (saenuruki) [inline components/Header/Header.jsx:11]: Hold on a moment! I'd love to hear everyone's thoughts on adding a background to the header...

**Comment by Jonnie-Dev** [comment]
Here is the preview video, sorry it's a lil bit blur, didnt remember to change the quality in my recording software

https://github.com/MeltingHackTeam/CommunityWebsite/assets/73708569/198fd5e4-f01d-462a-9247-6a51d913c736

**Comment by saenuruki** [comment]
Thank you!! LGTM!

**Review by saenuruki — APPROVED** [review]

**Comment by Jonnie-Dev** [comment]
Yup, merge it ⚡

**Pull request merged**